tests/RobotFramework/tests/mqtt/connect_broker.robot

```diff
@@ -10,25 +10,45 @@ Force Tags      theme:mqtt
 *** Test Cases ***
 Publish on a local insecure broker
     Start Service      mosquitto
-    Execute Command    tedge mqtt pub topic message
+    Execute Command    tedge mqtt pub topic1 message1
```

Comment:
I would add a check that the message is actually received - here and for the two other tests using authentication

Reply:
To verify that the message was received, I was thinking of starting `tedge mqtt pub` and `tedge mqtt sub` commands, but as `tedge sub` blocks, I don't really know how should I handle it using our ThinEdgeIO Robot Framework library. Could you give me some pointers? Because from the glimpse into the library, I don't see any way to start a background process on the test device.

Reply:
@Bravo555 Sorry I'm a bit late...There is a mqtt logger which is running in the integration test device where you can use the `Should Have MQTT Messages` keyword. The mqtt logger is running all the time, so no need to explicitly do a mqtt sub.

Here is an example:

```
@{listen}=    Should Have MQTT Messages    topic=tedge/${CHILD_SN}/commands/req/config_update        date_from=-5s
```